### PR TITLE
Hotel requests form fix

### PR DESCRIPTION
**Pull request by EliAndrewC**
There were two bugs with the hotel form on the volunteer checklist.  I've fixed both of them, which I'll explain inline.

### CODE REVIEW

File: uber/site_sections/signups.py [hunk 1]
```diff
@@ -81,6 +81,7 @@ def hotel_requests(self, session, message='', decline=None, **params):
             raise HTTPRedirect('index?message={}', 'Only Staffers can request hotel space')
         requests = session.hotel_requests(params, checkgroups=['nights'], restricted=True)
         if 'attendee_id' in params:
```
Comment (EliAndrewC) [inline uber/site_sections/signups.py]: One change we made this year was adding the ``admin_only`` parameter to the ``Column`` function in ``models.py``, which had the side effect of making it so we don't allow attendees to alter foreign keys anymore.  This actually makes sense; now that I think about it, someone who knew another attendee's id could have theoretically submitted or even edited their hotel requests until now!

File: uber/site_sections/signups.py [hunk 2]
```diff
@@ -102,11 +103,11 @@ def hotel_requests(self, session, message='', decline=None, **params):
         day_before = (c.EPOCH - timedelta(days=1)).strftime('%A')
         last_day = c.ESCHATON.strftime('%A').upper()
         day_after = (c.ESCHATON + timedelta(days=1)).strftime('%A')
```
Comment (EliAndrewC) [inline uber/site_sections/signups.py]: We previously just grabbed config out of the dictionary of global variables.  Now we do the more sensible ``getattr`` on our single global ``c`` object.

**Comment by binary1230** [comment]
looks good to me +1

**Comment by kitsuta** [comment]
+1

Dom's in the middle of the puppet refactor, we should merge this when that's done and over with.

**Pull request merged**